enforcer/src/main/java/org/wso2/choreo/connect/enforcer/util/FilterUtils.java

```diff
@@ -509,6 +510,18 @@ public static String getClientIp(Map<String, String> headers, String knownIp) {
         return clientIp;
     }
 
+    public static String getAuthHeaderName(RequestContext requestContext) {
+        AuthHeaderDto authHeader = ConfigHolder.getInstance().getConfig().getAuthHeader();
+        String authHeaderName = requestContext.getMatchedAPI().getAPIConfig().getAuthHeader();
+        if (StringUtils.isEmpty(authHeaderName) && !StringUtils.isEmpty(authHeader.getAuthorizationHeader())) {
```

Comment:
To avoid duplicate two checks in L516,  L519, shall we change the logic first check empty authHeaderName after check authHeader.getAuthorizationHeader() ?